earthaccess/exceptions.py

```diff
@@ -0,0 +1,13 @@
+class LoginStrategyUnavailable(Exception):
+    """The selected login strategy was skipped.
+
+    This should be raised when a login strategy can't be attempted, for example because
+    "environment" was selected but the envvars are not populated.
+
+    DO NOT raise this exception when a login strategy is attempted and failed, for
+    example because credentials were rejected.
+    """
+
+
+class LoginAttemptFailure(Exception):
+    """The login attempt failed."""
```

Comment:
To avoid potential confusion with nothing but a docstring, I suggest adding `pass`:
```suggestion
    """The login attempt failed."""
    pass
```

Reply:
Same as above :)